docs/src/makielayout/tutorial.md

```diff
@@ -532,3 +532,99 @@ documentation for more details and other dynamic parts like sliders and buttons!
 using GLMakie
 GLMakie.activate!()
 ```
+
+## Final code
+Here is the code producing the final figure above:
```

Comment:
Maybe add:

"Note that this code is not perfect, but simply reflects the tutorial content including unnecessary intermediate steps like misplacing and then moving the legend."

Reply:
Not sure what you mean by the unnecessary intermediate steps, can you elaborate?

Reply:
Well I deliberately place the legend in the "wrong" place first to show that you can move it, but of course if you run the whole script at once it's a bit silly to include that. But it's not supposed to be streamlined code, just the collection of all steps

Reply:
now you removed that step, but I wouldn't want the code down there to be different. just tell people that it's not how the "ideal" code would be written. it's the same with the heatmap example, they are also moved after creation.

Reply:
it's a kind of tutorial where some less than ideal things are shown first and then resolved